metis-debias/metis-debias-detect-service/src/main/java/eu/europeana/metis/debias/detect/service/DetectService.java

```diff
@@ -0,0 +1,19 @@
+package eu.europeana.metis.debias.detect.service;
+
+import eu.europeana.metis.debias.detect.model.DeBiasResult;
+import eu.europeana.metis.debias.detect.model.request.DetectionParameter;
+
+/**
+ * Implementations of this interface are able to detect biased terms given the languages
+ * and the terms to search, and it returns a report indicating the result of the terms.
+ */
+public interface DetectService {
```

Comment:
I would call this `BiasDetectionService` or something like that... certainly mentioning the word (de)bias. This is the name that calling code is mainly going to use, so I think it would be readable for calling code (outside the scope of this module).